plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/mapping/SeoMappingDAO.java

```diff
@@ -45,9 +46,21 @@
 
 	private static final Logger _logger =  LoggerFactory.getLogger(SeoMappingDAO.class);
 
+	private static final String ADD_MAPPING = 
```

Comment:
Does this require a migration to the database for existing installs?

Reply:
No, it not necessary